server/main_test.go

```diff
@@ -168,3 +168,87 @@ func TestStartGameRoute(t *testing.T) {
 	actualResponse := w.Body.String()
 	assert.Equal(t, expectedState, actualResponse)
 }
+
+func TestAddPromptRoute(t *testing.T) {
```

Comment:
Ok this test is where I draw the line, I'll create an issue to refactor tests after I'm done with the client-side drawing code

(No need to change anything, just voicing out my thoughts 😛)

Reply:
https://github.com/danbarragan/drawydraw/issues/22

Reply:
Haha I thought this might push you over the edge :)